List/list.css

```diff
@@ -0,0 +1,113 @@
+:root{
+    --special-color:black;
+    --border: 1px solid black;
+    --border-radius:1rem;
+    --special-padding:1rem;
+    --special-margin:0.5rem;
+    --special-margin-top:1rem;
+    --font-size:1.5rem;
+    --wt-color:rgb(53, 156, 53);
+    --fb-color:rgb(54, 54, 206);
+    --tr-color:rgb(33, 147, 192);
+    --dc-color:rgb(177, 172, 172);
+    --white-color:white;
```

Comment:
you can use rgb code here too for consistency

Reply:
ok will do that